src/Systmm.f90

```diff
@@ -13,23 +12,23 @@ Module SYSTM
 real   :: x,x_bndry,xd,xdd,xd_year,x_head,xwpd,year
 real,dimension(:),allocatable:: T_head,T_smth,T_trib
 real,dimension(:,:,:),allocatable:: temp
-!
-!
+!
+!
 logical:: DONE
 !
 ! Indices for lagrangian interpolation
 !
-integer:: npndx,ntrp
+integer:: npndx,ntrp
 integer, dimension(3):: ndltp=(/-2,-3,-3/)
 integer, dimension(3):: nterp=(/3,4,3/)
 
-!
+!
 real, parameter:: pi=3.14159,rfac=304.8
```

Comment:
something like `pi` should be defined in a common place (a module or file that gets included by all code), to avoid having multiple versions of the same constant.